ts/screens/profile/components/services/ServicesContactComponent.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import { View } from "native-base";
+import { FlatList, ListRenderItemInfo } from "react-native";
+import { useState } from "react";
+import { GlobalState } from "../../../../store/reducers/types";
+import { Dispatch } from "../../../../store/actions/types";
+import { H1 } from "../../../../components/core/typography/H1";
+import { Body } from "../../../../components/core/typography/Body";
+import { IOStyles } from "../../../../components/core/variables/IOStyles";
+import { H4 } from "../../../../components/core/typography/H4";
+import IconFont from "../../../../components/ui/IconFont";
+import { IOColors } from "../../../../components/core/variables/IOColors";
+import ItemSeparatorComponent from "../../../../components/ItemSeparatorComponent";
+import { H5 } from "../../../../components/core/typography/H5";
+import TouchableDefaultOpacity from "../../../../components/TouchableDefaultOpacity";
+import I18n from "../../../../i18n";
+
+type Props = {
+  hasAlreadyOnboarded?: true;
```

Comment:
Is this for future use? It is currently never used.

Reply:
Yep, it will be needed for this scenario:

<img src="https://user-images.githubusercontent.com/3959405/123272201-d333c100-d501-11eb-931d-4aa34ccf9724.png" height="550"/>

It will be shown fo old users opening app after the complete flow is released